Source/WebKit/NetworkProcess/curl/WebSocketTaskCurl.cpp

```diff
@@ -203,6 +203,11 @@ void WebSocketTask::didReceiveData(WebCore::CurlStreamID, const WebCore::SharedB
             if (!sendFrame(WebCore::WebSocketFrame::OpCodePong, data, length))
                 didFail("Failed to send WebSocket frame."_s);
             break;
+
+        case WebCore::WebSocketFrame::OpCodeContinuation:
+        case WebCore::WebSocketFrame::OpCodePong:
+        case WebCore::WebSocketFrame::OpCodeInvalid:
+            break;
```

Comment:
I don't have a strong opinion here.  How about an idea of using `default` here?

Reply:
If a new enumerator is added, we can get a new error and can figure out if we want to handle that case or not.

Reply:
Sounds good.